job-dsls/jobs/compile_downstream_build.groovy

```diff
@@ -0,0 +1,206 @@
+/**
+ * Creates downstream pullrequest (PR) jobs for kiegroup GitHub org. units.
+ * These jobs execute the downstream build skipping tests and skipping the gwt compilation
+ * for a specific PR to make sure the changes do not break the downstream repos.
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+def final DEFAULTS = [
+        ghOrgUnit              : Constants.GITHUB_ORG_UNIT,
+        branch                 : Constants.BRANCH,
+        timeoutMins            : 600,
+        label                  : "kie-rhel7 && kie-mem8g",
+        ghAuthTokenId          : Constants.GITHUB_AUTH_TOKEN,
+        upstreamMvnArgs        : "-B -e -T1C -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Drevapi.skip=true clean install",
+        downstreamMvnGoals     : "-B -e -nsu -fae clean install -Dfull=true -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true",
```

Comment:
Since we won't be running any tests in these "just compile" jobs, I would suggest adding -T1C into the downstreamMvnGoals - here maven parallelism shouldn't hurt, on the contrary it will make the jobs run faster.

Reply:
+1; adding -T1C seems to be a very good idea.